docs/en/writing-running-appium/tutorial/swipe/ios-picker-wheels-set-value.md

```diff
@@ -0,0 +1,33 @@
+## iOS pickerWheels: Fast '.setValue()' example
+
+PickerWheels in XCTest framework is powerful and weak side. Powerful
```

Comment:
Unfortunately, XCTest cannot always properly interact with picker wheel controls. Sometimes the `setValue()` call might not have any effect.